src/databases/Blueprint/Blueprint.xml

```diff
@@ -10,6 +12,8 @@
       ${CONDUIT_LIBRARY_DIR}
     </LDFLAGS>
     <LIBS>
+      avtmfem
+      avtblueprint 
```

Comment:
These won't be built if MFEM/Conduit not found, correct?
Then a CMake var should be used in place of the actual target name, so it can be empty if MFEM or Conduit isn't found. Same goes for the includes.

Same goes for MFEM.xml

Reply:
Those libs wont be created, however the plugins won't be built either. 

I don't think this is a problem given this?

(other areas are guarded)

Reply:
Of course, you are correct. My apologies.